prac_05/word_occurrences.py

```diff
@@ -0,0 +1,37 @@
+def main():
+    """get some text and display the amount of times a word was used"""
+    text = input("input string to count word occurrences\n>>>").split()
+    words = count_occurrences(text)
+    longest_word = get_longest_word(words)
+    list_words(words, longest_word)
+
+
+def get_longest_word(words):
+    """get the longest word"""
+    longest_word = ""
+    for word in words.keys():
+        if len(word) > len(longest_word):
+            longest_word = word
+    return longest_word
+
+
+def list_words(words, longest_word):
+    """list words and values"""
+    dict_items = sorted(words.items())
```

Comment:
`dict_items` is a poor name. This variable doesn't need to exist - use the value in the loop below.